.gitignore

```diff
@@ -21,3 +21,6 @@ vendor
 
 # Avoid pushing .gz files created during rsync
 *.gz
+_posts/de/newsletters/fragen.txt
+_posts/de/newsletters/Korrektorat.txt
+scripts/check_text_issues.py
```

Comment:
@Walpurga03 can you omit these changes from the PR?

Reply:
@Walpurga03 ping on this one

Reply:
They had already been removed, but now the `.gitignore` file was deleted. Will readd it.